examples/howto/layouts/manual_grid.py

```diff
@@ -27,12 +27,14 @@
 paragraph = Paragraph(text="We build up a grid plot manually. Try changing the mode of the plots yourself.")
 
 select = Select(title="Sizing mode", value="fixed", options=list(SizingMode), width=300)
+
+grid = GridBox(children=[(p1, 0, 1), (p2, 1, 0), (p3, 1, 1), (p4, 1, 2)])
```

Comment:
I would also use `grid([[None, p1, None], [p2, p3, p4]])` instead of the low-level API.